frontend/src/components/ContactList.vue

```diff
@@ -15,56 +15,60 @@
       <v-col cols="12" sm="12">
         <v-btn-toggle v-model="activeCircle" class="ml-4">
           <v-btn v-for="circle in circles" :key="circle" @click="filterByCircle(circle)"
-            :class="{ active: activeCircle === circle }">
+            :class="{ 'active-circle': activeCircle === circle }">
             {{ circle }}
           </v-btn>
-          <v-btn @click="clearCircleFilter" :class="{ active: activeCircle === null }">All</v-btn>
+          <v-btn @click="clearCircleFilter" :class="{ 'active-circle': activeCircle === null }">
+            All
+          </v-btn>
         </v-btn-toggle>
       </v-col>
     </v-row>
 
-   <!-- Contact Cards -->
-<v-row>
-  <v-col
-    v-for="contact in filteredContacts"
-    :key="contact.ID"
-    cols="12"
-    sm="6"
-    md="4"
-    lg="3"
-  >
-    <v-card
-      class="contact-card"
-      outlined
-      elevation="2"
-      @click="goToContact(contact.ID)"
-    >
-      <v-card-text>
-        <!-- Profile Photo -->
-        <v-avatar size="80" class="mb-3">
-          <v-img :src="`${backendURL}/contacts/${contact.ID}/profile_picture.jpg`" alt="Profile Photo"></v-img>
-        </v-avatar>
-
-        <!-- Contact Name -->
-        <div class="contact-name">
-          {{ contact.firstname }} {{ contact.lastname }}
-        </div>
-
-        <!-- Circles with Wrapping -->
-        <div class="circle-chips mt-2">
-          <v-chip
-            v-for="circle in contact.circles"
-            :key="circle"
-            class="mr-2 mb-2"
-            @click.stop="filterByCircle(circle)"
-          >
-            {{ circle }}
-          </v-chip>
-        </div>
-      </v-card-text>
-    </v-card>
-  </v-col>
-</v-row>
+    <!-- Display Current Search Query -->
+    <v-row v-if="searchQuery && searchQuery.trim()" class="mb-4">
+      <v-col cols="12">
+        <v-alert type="info" border="start" class="d-flex align-center" @click="clearSearch">
+          Showing results for: <strong>"{{ searchQuery }}"</strong>
+        </v-alert>
+      </v-col>
+    </v-row>
+
+    <!-- Contact Cards -->
+    <v-row>
+      <v-col v-for="contact in filteredContacts" :key="contact.ID" cols="12" sm="6" md="4" lg="3">
+        <v-card class="contact-card" outlined elevation="2" @click="goToContact(contact.ID)">
+          <v-card-text>
+            <!-- Profile Photo -->
+            <v-avatar size="80" class="mb-3">
+              <v-img :src="`${backendURL}/contacts/${contact.ID}/profile_picture.jpg`" alt="Profile Photo"></v-img>
+            </v-avatar>
+
+            <!-- Contact Name -->
+            <div class="contact-name">
+              {{ contact.firstname }} {{ contact.lastname }}
```

Comment:
In case of internationalisation, this should be expressed as a function (some cultures put the family name first).

Reply:
Great point! I would shelf this now and instead put it on a feature list that we write some international functions available for all components (like ReturnFullName() in this case). We will need to use them for ContactView, ActivityAdd (select contact) and RelationshipList as well.